content/install_maintain/installation/installation-modes.md

```diff
@@ -0,0 +1,58 @@
+
+* [Installing and Configuring a Cloudify Manager]({{< relref "install_maintain/installation/installing-manager" >}})
+* [Deploying a Cloudify Manager Image]({{< relref "install_maintain/installation/manager-image" >}})
+
+
+## Cloudify Compact cluster (3 nodes)##
+
+The Cloudify compact cluster deployment is based on 3 servers, each running all of the Cloudify services, where the services are deployed in an active-active approach with a high-availability setup.
+A Compact cluster provides an enterprise grade Cloudify deployment suitable to almost any organization, with a performance and scale equivalent to a fully distributed cluster containing 3 managers and with the option of further scaling using higher form factor hardware. The compact cluster is designed for mission critical use.
+
+To learn more about installing the Cloudify Compact Cluster please continue to:
+
+* [Cloudify Compact cluster installation guide]({{< relref "install_maintain/installation/three-nodes-cluster" >}})
+
+{{% note %}}  
+Note! The above mentioned page will guide you how to manually deploy a Compact cluster. Cloudify provides an automated flow that simplifies the deployment over three existing VMs. To use the Cloudify Cluster Manager and automate the cluster deployment flow read: [Cloudify Cluster Manager package] ({{< relref "install_maintain/installation/cfy-cluster-manager.md" >}})
+{{% /note %}}
+
+
+## Cloudify fully distributed cluster ##
+
+The Cloudify platform can be deployed as a fully distributed cluster, running each of the Cloudify services (mainly the manager, the database, and the messaging queue) on a dedicated VM/node. This mode of installation provides the complete flexibility to deploy the database and the messaging queue in a highly available mode (3 VMs per each service), or leverage external services for them in the form of an external service or a public managed service such as Amazon RDS. It further allows scaling leveraging 3 or more manager servers in an active-active mode.
+
+Running the Cloudify platform as a fully distributed cluster provides the most flexibility and scaling options and fits any enterprise for mission critical use.
+
+To learn more about installing the Cloudify fully distributed Cluster please continue to:
+
+* [Cloudify fully distributed cluster installation guide]({{< relref "install_maintain/installation/installing-cluster" >}})
+* [Distributed Cluster with External Database and Messaging queue]({{< relref "install_maintain/installation/installing-external-db-and-queue-cluster" >}})
+
+{{% note %}}  
+Note! The above mentioned page will guide you how to manually deploy a Cloudify cluster. Cloudify provides an automated flow that simplifies the deployment over three existing VMs. To use the Cloudify Cluster Manager and automate the cluster deployment flow read: [Cloudify Cluster Manager package] ({{< relref "install_maintain/installation/cfy-cluster-manager.md" >}})
+{{% /note %}}
```

Comment:
It might be worth adding a comment about the load-blancer somewhere in this page.